.storybook/foundations/corner-rounding/action-button-corner-rounding.stories.js

```diff
@@ -0,0 +1,111 @@
+import { html } from "lit";
```

Comment:
I really don't like this approach of adding separate stories for Checkbox and ActionButton but as the [guidance on this in the Slack thread](https://adobedesign.slack.com/archives/GQ09RCBA5/p1708130655322169?thread_ts=1708116176.602529&cid=GQ09RCBA5) stated:

> Component size tokens:
small – four checkboxes in small, medium, large, extra-large using these tokens
medium – five action buttons in extra-small, small, medium, large, extra-large using these tokens

This does technically fulfill that requirement, but I'm going to keep looking to see if there's a less awkward approach to get there.

Reply:
We chatted about this in dev huddle, decided to go with [this API solution](https://github.com/storybookjs/storybook/issues/9209#issuecomment-1866309500) to hide the documentation stories from the sidebar. We should probably still check if Chromatic sees these...